app/components/Mods/ModTableRow.tsx

```diff
@@ -181,6 +187,14 @@ const ModTableRow: React.FunctionComponent<Props> = ({ mod }) => {
     if (isModConflicting) {
       return modsText.conflictingModWarning(conflictingMods.join(', '));
     }
+    if (isInstalled(mod) && wrongBepinExVersion){
+      if (!bepInEx || !isInstalled(bepInEx)) {
+        return modsText.missingDependencyWarning("bbepis.BepInEx");
```

Comment:
the `bbepis.BepInEx` string is showing in a lot of places. Better move it to some separate file with uniqueName constants, together with `Locochoco.CMOWA` and `Alek.OWML` (this last one not your fault but might as well do it now)